packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -321,11 +327,11 @@ class OverflowMenu extends Component {
     }
   };
 
-  closeMenu = () => {
-    let wasOpen = this.state.open;
+  closeMenu = (cb) => {
```

Comment:
is this change also intentional? the `closeMenu` method is internal and I don't see any new callback usage

Reply:
Yep, it's being called on line 310 with the callback, moving the call to `focusMenuEl` to be executed only on Escape press, which I believe was the desired behaviour.